components/canvas/webgl_paint_task.rs

```diff
@@ -184,6 +190,16 @@ impl WebGLPaintTask {
         chan.send(pixels).unwrap();
     }
 
+    fn send_native_surface(&self, _: Sender<NativeSurface>) {
+        unimplemented!()
+        // We need to make a clone of the surface
+        // let surface = self.gl_context.borrow_draw_buffer().unwrap()
+        //                             .borrow_bound_surface().unwrap();
+        // let surface = (*surface).clone();
+        // surface.mark_wont_leak();
+        // chan.send(surface).unwrap()
```

Comment:
I'd rather not have a bunch of commented code around—it tends to bitrot quickly. Could you remove this?